app/views/channels/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with(model: channel, local: true) do |form| %>
+  <% if channel.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(channel.errors.count, "error") %> prohibited this channel from being saved:</h2>
+
+      <ul>
+      <% channel.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= form.submit %>
+  </div>
+<% end %>
```

Comment:
Please remove all views. We're only implement API as the requirements. It should only return json data.

Reply:
Now , it only return json data . So the view dont influence to project. ok. it is removed now.